.github/workflows/nightly-build.yaml

```diff
@@ -38,20 +56,20 @@ jobs:
       - name: Set up Docker Buildx
         uses: docker/setup-buildx-action@v1
       - name: Login to DockerHub
-        if: github.event_name != 'pull_request' && github.repository_owner == 'kubesphere'
+        if: github.repository_owner == 'kubesphere'
```

Comment:
GitHub Actions do not allow to read the `secrets` if it's a PR. So, I guess that we still need the following condition.

`github.event_name != 'pull_request'`

Reply:
I think this NightlyBuild workflow only trigger by cron (schedule type event).
PR won't trigger this workflow.

Reply:
Fair enough.